app/controllers/api/v1x0/workflows_controller.rb

```diff
@@ -76,6 +73,27 @@ def rbac_scope(relation)
 
         relation
       end
+
+      def collection(base_query)
+        resp = ManageIQ::API::Common::PaginatedResponse.new(
+          :base_query => filtered(scoped(base_query)),
+          :request    => request,
+          :limit      => params[:limit],
+          :offset     => params[:offset]
+        ).response
+
+        json_response(resp)
+      end
+
+      def resource_object_params
+        @resource_object_params ||= params.slice(:object_type, :object_id, :app_name).to_unsafe_h
```

Comment:
where is `resource_object_query_params` used? shouldn't it be `params[:resource_object_query_params].slice()`? same in the spec test.